gitcoinco/grants-stack
# manager: Create programs for allo v1 and v2

Fixes: #2976 
## Description

ended up mostly deleting code 🙌 

- add createProgram allo method
- implement create program for v1 (program factory) v2 (create profile)
- add `type: program` to the metadata so we can distinguish projects from programs
- Remove create program context in favour of simple hook
- remove context unit test in favour of create program page integration test

# How to test

Loom: https://www.loom.com/share/2ea73c01bb09415a97f6e22a083b822b

Note: Profiles created in Builder will not show up in manager anymore.

1. Create a program in Allo V2, it should show up, try with multiple operators
2. Repeat for Allo V1




## Checklist

This PR:
